draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -204,11 +207,22 @@ for authoritative nameserver management only and are not intended for general
 querying via recursive resolvers.
 
 
-## Global Properties
+## Properties
 
 Apart from catalog zone metadata stored at the apex (NS, SOA and the like), catalog zone information is stored in the form of "properties".
 Catalog consumers SHOULD ignore properties they do not understand.
 
+Properties have a name combined with an associated RR type.
+Properties that have a name known to a catalog consumer, but the wrong RR type,
+SHOULD be ignored.
+
+Known properties with the correct RR type, but which are for some reason
+invalid (for example because of an impossible value or because of an illegal
+number of RRs in the RRset), denotes a broken catalog zone which MUST NOT be
+processed (see (#generalrequirements)).
+A clear error message stating which invalid property caused the broken catalog,
+with the reason why it was considered invalid, MUST be logged.
```

Comment:
@Habbie commented on "logged" above; ditto here.

Reply:
Personally I see logging as something that is not speficically required to make catalog zones interoperable with another, so I would be in favor of using non-RFC2119 language here.

Reply:
It is actually relevant for interoperability because properties are optional and thus consumers may behave differently with "broken" properties (i.e. stop processing or ignore)

Reply:
Fixed in commit Aram Sargsyan

Reply:
The text has changed to:

The reason a catalog zone is considered broken SHOULD always be communicated clearly to the operator (e.g. through a log message).

@matje Do you feel "SHOULD" should be "should"?

Reply:
I guess SHOULD is just fine...